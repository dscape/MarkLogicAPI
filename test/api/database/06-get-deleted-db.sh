#!/bin/sh
curl -sv localhost:6984/mapitest 2>&1 \
  | sed -e 's/^\*.*//' \
  | grep '[<|{].[A-C|E-Z|a-z]'