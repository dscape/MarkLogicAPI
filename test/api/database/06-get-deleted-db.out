< HTTP/1.1 404 Not Found
< Cache-Control: must-revalidate
< Content-type: text/plain; charset=UTF-8
< Server: MarkLogic
< Content-Length: 57
< Connection: close
{"error":"not_found","reason":"Database does not exist."}
